x-pack/plugins/infra/public/components/waffle/waffle_node_type_switcher.tsx

```diff
@@ -24,7 +25,15 @@ export class WaffleNodeTypeSwitcher extends React.PureComponent<Props> {
   public render() {
     return (
       <EuiKeyPadMenu>
-        <EuiKeyPadMenuItem label="Hosts" onClick={this.handleClick(InfraNodeType.host)}>
+        <EuiKeyPadMenuItem
+          label={
+            <FormattedMessage
+              id="xpack.infra.waffle.nodeTypeSwitcher.hostsLabel"
+              defaultMessage="Hosts"
+            />
+          }
+          onClick={this.handleClick(InfraNodeType.host)}
+        >
           <img src="../plugins/infra/images/hosts.svg" className="euiIcon euiIcon--large" />
```

Comment:
"Kubernetes" and "Docker" are not translated, which is correct. I think "Hosts" shouldn't be localized either.

Reply:
@skh Could you please write this terms, which shouldn't be translated, in out Glossary doc
https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit

Reply:
@tibmt Done.

Reply:
After discussion this with Jin I've removed the terms for the Infrastructure UI from the glossary, and this PR is good to go.